src/content/impact/think/think.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Think
+---
+
```

Comment:
Would update this to something like this.

```
<grid classname="background--nopadding">
<column lg="16">

<p><icon color="blue" name="ArrowLeft20" inline="true"></icon>  <a href="/Impact"><span>Impact Stories</span></a></p>

</column>
</grid>
<grid classname="background--bleed">
<column lg="16">

<img alt="add alt" src="./images/Impact_1.png">

</column>
</grid>
```

and then add this css to the grid scss file
```
.background--bleed {
  padding: 0;

  .ibm--grid {
    padding: 0;
  }

  .ibm--col-lg-16 {
    padding: 0;
  }
}

.background--nopadding {
  padding: 1.5rem 0 0 0;
}
```